plat/qemu/platform.mk

```diff
@@ -4,6 +4,24 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
+ifeq (${ARM_ARCH_MAJOR},7)
+# Qemu expects a BL32 boot stage.
+NEED_BL32		:=	yes
+# Qemu Cortex-A15 model does not implement the virtualization extention.
+# For this reason, we cannot set ARM_CORTEX_A15=yes and must define all
+# the ARMv7 build directives.
+MARCH32_DIRECTIVE 	:= 	-mcpu=cortex-a15
+$(eval $(call add_define,ARMV7_SUPPORTS_LARGE_PAGE_ADDRESSING))
+$(eval $(call add_define,ARMV7_SUPPORTS_GENERIC_TIMER))
+endif # ARMv7
+
+ifeq (${SPD},opteed)
+add-lib-optee 		:= 	yes
```

Comment:
This would be to cover the AArch64 mode. Would be worh to call this out in the commit message.

Reply:
This does not change the previous implementation which already integrate the optee lib for AArch64 based on `ifeq (${SPD},opteed)`.

Reply:
OK.